admin/config/feature-toggles-Azure_UAT.json

```diff
@@ -1,4 +1,7 @@
 {
   "groupCreate": false,
-  "groupRemove": false
+  "groupRemove": false,
```

Comment:
at the risk of sounding pedantic, can we use a dot, rather than a comma in the filenames please?

Reply:
Yes, sure.